lib/v2/screens/transfer/send/send_confirmation/interactor/viewmodels/send_confirmation_arguments.dart

```diff
@@ -2,6 +2,12 @@ class SendConfirmationArguments {
   final String account;
   final String name;
   final Map<String, dynamic> data;
+  final int pops;
```

Comment:
what is pops? Why do we need this?

Reply:
We need pops because the screen is shown from different screens, sometimes we need 2 pop() calls, sometimes 3